git/git_test.go

```diff
@@ -510,7 +510,7 @@ func (gitSuite *gitTestSuite) TestSnapshotWithIndexPath() {
 	})
 
 	gitSuite.Nil(err)
-	gitSuite.verifyIndexFile(40, indexFilePath)
+	gitSuite.verifyIndexFile(41, indexFilePath)
```

Comment:
Is it 41 cause you added a file with special characters in the test repo?

Reply:
nope. because of the headers line...

Reply:
Might be a good idea to add a test for it then

Reply:
done

Reply:
I meant a test for the special characters

Reply:
this requires adding these files to dc-heacth github.com/apiirolab (the repo that the test is running on).
I don't have permissions to, and I am not sure what this could break...
we can also get back to this later.